spark3/src/test/java/org/apache/iceberg/actions/TestRemoveOrphanFilesAction3.java

```diff
@@ -19,5 +19,46 @@
 
 package org.apache.iceberg.actions;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+import org.apache.iceberg.spark.SparkCatalog;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.spark.source.SparkTable;
+import org.apache.spark.sql.catalyst.analysis.NoSuchTableException;
+import org.apache.spark.sql.catalyst.analysis.TableAlreadyExistsException;
+import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.connector.expressions.Transform;
+import org.junit.Assert;
+import org.junit.Test;
+
 public class TestRemoveOrphanFilesAction3 extends TestRemoveOrphanFilesAction {
+  @Test
```

Comment:
I only added this, (and the test inside orphan files) because we'll have additional coverage for the use cases once the SQL api is introduced and vectorizing the entire suite of actions was getting a bit onerous.